habana/ubi8-python-3.8/Dockerfile

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) 2022 Habana Labs, Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# HabanaLabs Dockerfile base installer layer for RedHat 8.6
```

Comment:
Can we attach the link as well?
https://github.com/HabanaAI/Setup_and_Install/blob/1.9.0/dockerfiles/Dockerfile_rhel8.6_base_installer

Reply:
surely :+1: